packages/block-editor/src/hooks/typography.js

```diff
@@ -250,3 +247,31 @@ function useIsTypographyDisabled( props = {} ) {
 
 	return configs.filter( Boolean ).length === configs.length;
 }
+
+/**
+ * Returns an object containing default controls. A control key with a `true` value
+ * means that the control will be shown in the panel by default.
+ *
+ * @param {string|Object} blockType Block name or block type object.
+ *
+ * @return {Object} Default controls key/value pairs.
+ */
+export function getDefaultTypographyControls( blockType ) {
+	const defaultBorderControls = getBlockSupport( blockType, [
+		TYPOGRAPHY_SUPPORT_KEY,
+		'__experimentalDefaultControls',
+	] );
+
+	if ( defaultBorderControls === 'all' ) {
+		return {
```

Comment:
It looks like this is missing `textTransform`?

Reply:
Thanks for catching all the whacky copy/paste jobs @andrewserong 

Fixed.